selenium_tests/lti_emailer/permission_test.py

```diff
@@ -0,0 +1,70 @@
+__author__ = 'hic048'
+import unittest
+from ddt import ddt, data, unpack
+
+from selenium.common.exceptions import TimeoutException
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium_tests.base_test_case import get_xl_data
+from selenium_tests.lti_emailer.emailer_base_test_case import EmailerBaseTestCase
+from selenium_tests.lti_emailer.page_objects.emailer_permission_po import PermissionPage, Locator2
+from selenium_tests.lti_emailer.emailer_base_test_case import CANVAS_ADD_USERS
+
+
+class RoleTest (object):
+    _mailing_list_name = "Mailing List for E-PSCI 333"
+    _masquerade_url_base = "https://canvas.icommons.harvard.edu/users/"
+    _validate_denied_msg = "Validating denied access for course level roles/ Canvas user ID:"
+    _validate_access_msg = "Validating access for Canvas user ID:"
+
+@ddt
+
```

Comment:
you can remove this blank line so it's clearer that the decorator `@ddt` decorates the class `test_permission`